### PR TITLE
Fix exit codes in case of errors

### PR DESCRIPTION
**Pull request by kporowski**

### CODE REVIEW

File: src/cli-main.ts [hunk 1]
```diff
@@ -365,7 +360,7 @@ program
       await updateHostnamesAndRules(requestHostnames, rules, sandboxId, sandboxPropertyId);
       console.log(`Successfully updated sandbox_id: ${sandboxId} sandbox_property_id: ${sandboxPropertyId}`);
     } catch (e) {
-      console.log(e);
+      cliUtils.logAndExit(1, e);
```
Comment (krzyk) [inline src/cli-main.ts:363]: To be consistent, cases where logging just the exception should be prepended with `ERROR: `

File: src/cli-main.ts [hunk 2]
```diff
@@ -697,7 +692,7 @@ async function createFromRecipe(recipeFilePath, name, clonable, cpcode) {
   } else if (sandboxRecipe.cloneFrom) {
     res = await createFromCloneRecipe(recipe);
   } else {
-    logAndExit("Error: could not find either sandbox.properties or sandbox.cloneFrom.");
+    cliUtils.logAndExit(1, "ERROR: could not find either sandbox.properties or sandbox.cloneFrom.");
```
Comment (krzyk) [inline src/cli-main.ts:695]: should be single quote

File: src/cli-main.ts [hunk 3]
```diff
@@ -714,7 +709,7 @@ function createRecipeProperty(rp, sandboxId) {
   } else if (rp.hostname) {
     return addPropertyToSandboxFromHostname(sandboxId, rp.requestHostnames, rp.hostname);
   } else {
-    logAndExit("Critical error with recipe property. Define the rulesPath or property.");
+    cliUtils.logAndExit(1, "ERROR: Critical error with recipe property. Define the rulesPath or property.");
```
Comment (krzyk) [inline src/cli-main.ts:712]: should be single quote

File: src/cli-main.ts [hunk 4]
```diff
@@ -726,7 +721,7 @@ function createRecipeSandboxAndProperty(rp, propertyForRules, recipe, cpcode) {
   } else if (rp.rulesPath) {
     return createFromRules(rp.rulesPath, propertyForRules, rp.requestHostnames, recipe.clonable, recipe.name, cpcode);
   } else {
-    logAndExit("Critical error with recipe property. Define the rulesPath or property.");
+    cliUtils.logAndExit(1, "ERROR: Critical error with recipe property. Define the rulesPath or property.");
```
Comment (krzyk) [inline src/cli-main.ts:724]: should be single quote

File: src/cli-main.ts [hunk 5]
```diff
@@ -782,13 +777,13 @@ program
       let propForRules;
       //validation
       if (!isNonEmptyString(name)) {
-        logAndExit(`You must provide a name for your sandbox.`);
+        cliUtils.logAndExit(1, `ERROR: You must provide a name for your sandbox.`);
```
Comment (krzyk) [inline src/cli-main.ts:780]: also backticks should be single quote if there is no interpolotation

File: src/cli-main.ts [hunk 6]
```diff
@@ -956,7 +953,7 @@ program
           }
         });
         if(matchedLocalSandbox) {
-          logAndExit(`\nAborting Sync...\nThe sandbox with sandbox id : ${sandboxId} and sandbox name ${localMatchedSandboxName} is already synced locally. Further syncs are not required for further updates to this sandbox.`);
+          cliUtils.logAndExit(0, `\nAborting Sync...\nThe sandbox with sandbox id : ${sandboxId} and sandbox name ${localMatchedSandboxName} is already synced locally. Further syncs are not required for further updates to this sandbox.`);
```
Comment (krzyk) [inline src/cli-main.ts:956]: I'm not sure if we should keep multi-line error messages, they are not grep friendly

File: src/cli-main.ts [hunk 7]
```diff
@@ -8,11 +8,11 @@ const jwtDecode = require('jwt-decode');
 const CLI_CACHE_PATH = process.env.AKAMAI_CLI_CACHE_PATH;
 
 if (!CLI_CACHE_PATH) {
-  logAndExit("Error: AKAMAI_CLI_CACHE_PATH is not set.");
+  cliUtils.logAndExit(1, "ERROR: AKAMAI_CLI_CACHE_PATH is not set.");
```
Comment (krzyk) [inline src/cli-main.ts:11]: where are cliUtils defined?

I don't know how the logging is done, but to be compliant with CLIs we should report errors to stderr

File: src/cli-main.ts (hunk 6 above)
Comment (kporowski) [reply src/cli-main.ts:956]: Agree, I would like this to be addressed in #60

**Pull request merged**